service/src/main/java/uk/gov/hmcts/reform/fpl/enums/UserRole.java

```diff
@@ -20,4 +21,10 @@
     public List<String> getRoles() {
         return ImmutableList.of("caseworker", "caseworker-publiclaw", this.role);
     }
+
+    public static boolean isHmctsUser(String userRole) {
+        List<UserRole> hmctsRoles = ImmutableList.of(HMCTS_ADMIN, GATEKEEPER, JUDICIARY, HMCTS_SUPERUSER);
```

Comment:
Might be worth extracting to a private method on this model for re-use. getAllHmctsRoles()

Reply:
I think it's fine as is.. when i moved to private method its 5 lines instead of 1 line for not much benefit.